Mathlib/Topology/Algebra/Order/Compact.lean

```diff
@@ -193,6 +193,73 @@ theorem IsCompact.exists_isLUB [ClosedIciTopology α] {s : Set α} (hs : IsCompa
   IsCompact.exists_isGLB (α := αᵒᵈ) hs ne_s
 #align is_compact.exists_is_lub IsCompact.exists_isLUB
 
+theorem _root_.cocompact_le_atBot_atTop [LinearOrder α] [CompactIccSpace α] :
+    cocompact α ≤ atBot ⊔ atTop := by
+  refine fun s hs ↦ mem_cocompact.mpr <| (isEmpty_or_nonempty α).casesOn ?_ ?_ <;> intro
+  · exact ⟨∅, isCompact_empty, fun x _ ↦ (IsEmpty.false x).elim⟩
+  · obtain ⟨t, ht⟩ := mem_atBot_sets.mp hs.1
+    obtain ⟨u, hu⟩ := mem_atTop_sets.mp hs.2
+    refine ⟨Icc t u, isCompact_Icc, fun x hx ↦ ?_⟩
+    exact (not_and_or.mp hx).casesOn (fun h ↦ ht x (le_of_not_le h)) fun h ↦ hu x (le_of_not_le h)
+
+theorem _root_.cocompact_le_atBot [LinearOrder α] [OrderTop α] [CompactIccSpace α] :
+    cocompact α ≤ atBot := by
+  refine fun _ hs ↦ mem_cocompact.mpr <| (isEmpty_or_nonempty α).casesOn ?_ ?_ <;> intro
+  · exact ⟨∅, isCompact_empty, fun x _ ↦ (IsEmpty.false x).elim⟩
+  · obtain ⟨t, ht⟩ := mem_atBot_sets.mp hs
+    refine ⟨Icc t ⊤, isCompact_Icc, fun _ hx ↦ ?_⟩
+    exact (not_and_or.mp hx).casesOn (fun h ↦ ht _ (le_of_not_le h)) (fun h ↦ (h le_top).elim)
+
+theorem _root_.cocompact_le_atTop [LinearOrder α] [OrderBot α] [CompactIccSpace α] :
+    cocompact α ≤ atTop := by
+  refine fun _ hs ↦ mem_cocompact.mpr <| (isEmpty_or_nonempty α).casesOn ?_ ?_ <;> intro
+  · exact ⟨∅, isCompact_empty, fun x _ ↦ (IsEmpty.false x).elim⟩
+  · obtain ⟨t, ht⟩ := mem_atTop_sets.mp hs
+    refine ⟨Icc ⊥ t, isCompact_Icc, fun _ hx ↦ ?_⟩
+    exact (not_and_or.mp hx).casesOn (fun h ↦ (h bot_le).elim) (fun h ↦ ht _ (le_of_not_le h))
+
+private theorem _root_.cocompact_ge_aux {l : Filter α}
+    (h_aux : (s t : Set α) → t.Nonempty → Nonempty α → IsCompact t → tᶜ ⊆ s → s ∈ l) :
+    cocompact α ≥ l := by
```

Comment:
```suggestion
    cocompact α ≥ l := by
```
You should never use `≥` in mathlib (and you should never use `ge` in the statements either). Can you reformulate everything with the inequality swapped?